.github/workflows/update_daily_ dependency.yml

```diff
@@ -0,0 +1,34 @@
+name: Check Dependencies
```

Comment:
- The name of this `.yml` file has a space in it.
- Maybe instead call it something like `Update mypy and pyright` (and rename the `.yml` file accordingly)?